src/main/java/spoon/support/sniper/internal/AbstractSourceFragmentContextCollection.java

```diff
@@ -69,10 +69,10 @@ public boolean knowsHowToPrint(PrinterEvent event) {
 	}
 
 	@Override
-	protected Boolean isFragmentModified(SourceFragment fragment) {
+	protected ModificationStatus isFragmentModified(SourceFragment fragment) {
 		//we cannot fast detect if it is modified using our changeResolver.
 		//So return null. The code later will detect it including modified roles.
```

Comment:
update comment, null value is not returned anymore?

Reply:
done, thanks.